test/uk/gov/hmrc/customs/emailfrontend/views/components/FieldsetSpec.scala

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Could you please add below package statement at the top of the file and give a line break after that?

package uk.gov.hmrc.customs.emailfrontend.views.components